readme.md

```diff
@@ -68,6 +68,16 @@ This SDK also contains options you can set and platform specific features.  You
 | [Jobs](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-jobs)                                  | :x:                | Use your backend app to perform job operation.                                                                                     |
 | [File Upload](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-file-upload)                    | :x:                | Set up your backend app to send file upload notification receiver.                                                                 |
 
+### Provisioning client SDK
+This repository contains [provisioning client SDK](./provisioning_client) for the [Device Provisioning Service](https://docs.microsoft.com/en-us/azure/iot-dps/).  Provisioning service SDK for C is work in progress.  Please see provisoning service SDK in [Node.js](https://github.com/Azure/azure-iot-sdk-node) and [Java](https://github.com/Azure/azure-iot-sdk-java) in the meantime.
```

Comment:
Should there be a comment about the device SDK for C since the service SDK is called out explicitly? Though it looks like the matrix is only for the service SDK.

Reply:
This matrix only list device SDK feature.  You can take a look at the service SDK matrix here: https://microsoft.sharepoint.com/:x:/r/teams/Azure_IoT/_layouts/15/WopiFrame.aspx?sourcedoc=%7BC1152315-8EA6-4C5A-83E8-424BF6F2FA12%7D&file=Features%20support.xlsx&action=default

Reply:
Updated [Link](https://microsoft.sharepoint.com/:x:/r/teams/Azure_IoT/_layouts/15/WopiFrame.aspx?sourcedoc=%7BC1152315-8EA6-4C5A-83E8-424BF6F2FA12%7D&file=Features%20support.xlsx&action=default)

Reply:
Okay, I got confused because the section talks about the client SDK but not whether or not it's for the device client or service client.